DependencyInjection/Configuration.php

```diff
@@ -27,7 +27,9 @@ public function getConfigTreeBuilder()
         $rootNode->children()
                 ->booleanNode('enabled')->defaultFalse()->end()
                 ->scalarNode('cypher_method')->defaultNull()->end()
```

Comment:
add  the info, that the first method returned from 'openssl_get_cipher_methods(false)' will be used as default.